annotations/views/annotation_views.py

```diff
@@ -43,6 +44,11 @@ def relations(request):
 
     paginator = Paginator(qs, 40)
     page = request.GET.get('page')
+
+    gt = request.GET.copy()
+    if 'page' in gt:
+        del gt['page']
+
```

Comment:
is there a reason you're not using the filters? This doesn't solve the problem of printing what ever is passed to the method to the page without any validation.